book/02-git-basics/sections/viewing-history.asc

```diff
@@ -231,56 +231,56 @@ Nakita mo na ang opsyon na iyon -- ang `-2` na opsyon, na nagpapakita lamang sa
+ay naglilimit sa commit output upang ang mga commit na tumugma _sa anuman_ sa `--author` na mga palatandaan at _anuman_
+sa `--grep` na mga palatandaan; gayunpaman, pagdagdag sa `--all-match` na opsyon sa karagdagang mga limitasyon sa output sa 
+mga nai-commit lang na tumutugma _sa lahat _ `--grep` ng mga palatandaan.
 ====
 
-Another really helpful filter is the `-S` option (colloquially referred to as Git's ``pickaxe'' option), which takes a string and shows only those commits that changed the number of occurrences of that string.
-For instance, if you wanted to find the last commit that added or removed a reference to a specific function, you could call:
+Isa pang nakakatulong talaga na filter ay ang `-S` na opsyon (ang pananalita ay tinutukoy bilang Git's ``pickaxe'' na opsyon), na kung saan kay kumukuha ng isang string at nagpapakita lamang sa mga commit na iyon na nagbago ang bilang ng mga pangyayari ng string na iyon.
+Halimbawa, kung gusto mong humanap sa huling commit na idinagdag na o inalis ang isang reperensiya sa tiyak na pag-andar, maaari kang tumawag:
 
 [source,console]
 ----
 $ git log -S function_name
 ----
 
-The last really useful option to pass to `git log` as a filter is a path.
-If you specify a directory or file name, you can limit the log output to commits that introduced a change to those files.
-This is always the last option and is generally preceded by double dashes (`--`) to separate the paths from the options.
+Ang huling talagang kapaki-pakinabang na opsyon upang makapasa sa `git log` na bilang isang filter ay isang path.
+Kung ikaw ay tumukoy ng direktoryo o pangalan ng file, maaari mong limitahan ang log output para i-commit ang ipinakilala na pagbago ng mga file na iyon.
+Ito ay laging huling opsyon at sa pangkalahatan ay nauna ng dobleng mga gitling (`--`) upang ihiwalay ang mga path mula sa mga opsyon.
 
-In <<limit_options>> we'll list these and a few other common options for your reference.
+Sa <<limit_options>> kami ay maglista nito at ilan pang iba na pangkaraniwang mga opsyon para sa iyong reperensiya.
 
 [[limit_options]]
-.Options to limit the output of `git log`
+.Mga opsyon upang limitahan ang output sa `git log`
 [cols="2,4",options="header"]
 |================================
-| Option                | Description
-| `-<n>`                | Show only the last n commits
-| `--since`, `--after`  | Limit the commits to those made after the specified date.
-| `--until`, `--before` | Limit the commits to those made before the specified date.
-| `--author`            | Only show commits in which the author entry matches the specified string.
-| `--committer`         | Only show commits in which the committer entry matches the specified string.
-| `--grep`              | Only show commits with a commit message containing the string
-| `-S`                  | Only show commits adding or removing code matching the string
+| Option                | Paglalarawan
```

Comment:
Paglalarawan is Description.. this should be Opsyon o Pagpipilian

Reply:
done change.